development/extensions/tutorial_basics.rst

```diff
@@ -38,9 +38,9 @@ The extension name is the name of the extension. In this tutorial we will use ``
 
 .. important::
 
-    Both the vendor and extension names must start with a lower or upper case letter, followed by letters and numbers
-    only. **Underscores, dashes and other characters are not permitted.** It is perfectly fine to have an extension
-    named ``iamanextension``.
+    Both the vendor and extension names must be lowercase and must start with a letter. Names can contain numbers
```

Comment:
"must be lowercase" is incompatible with numbers, which is why the two sentences were joined before. This should probably be:
"...must start with a lowercase letter, followed by lowercase letters and numbers only."

Reply:
Updated